tests/cnot/base/optim_history.dat

```diff
@@ -0,0 +1 @@
+#iter    Objective           ||Pr(grad)||           LS step           1-F_avg         Tikhonov-regul      Penalty-term
```

Comment:
This file is empty and should be removed. Please also check for other empty files.

Reply:
@steffi7574 Thank you for discovery. I am wondering why it is empty although it does compute gradient. I will look into this.

Reply:
Not sure what happened, but now it generates non-empty optim_history.dat files, so I have included them.